platforms/hyperledger-besu/charts/node_orion/templates/deployment.yaml

```diff
@@ -120,16 +120,16 @@ spec:
 
           vault_secret_key="${VAULT_SECRET_ENGINE}/${VAULT_SECRET_PREFIX}/{{ .Values.vault.keyname }}"
           vaultBevelFunc 'readJson' ${vault_secret_key}
-          validateVaultResponseHashicorp "${vault_secret_key}" "${VAULT_SECRET}" "LOOKUPSECRETRESPONSE" 
```

Comment:
why is this removed? I thought this is what Ravi added

Reply:
Yes, just shifted this function within the `bevel-vault.sh` script itself.  Validation will be handled by `bevel-vault.sh` itself, there is no need to define function `validateVaultResponseHashicorp` separately in each different chart.

Reply:
the sprint handles error checking and validation for the responses.